drapps/publish.py

```diff
@@ -60,8 +60,9 @@ def publish(
     source_application: Optional[str],
     skip_wait: bool,
 ) -> None:
-    """Updates a custom app, this covers the basic case of a new name
-    and also the custom apps publishing work which is scoped for 10.2
+    """Updates the source version of custom app and triggers the app to restart with
```

Comment:
```suggestion
    """Updates the source version of the custom app and triggers the app to restart with
```
Should there be `a/the` here?